style.css

```diff
@@ -464,6 +570,39 @@
     margin-top: 20px;
   }
 
+  .modal-btn2 {
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+  }
+
+  .modal-btn1 {
+    display: none;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+  }
+
+  .modal-btn a {
+    text-decoration: none;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    gap: 12px;
+    width: 130px;
+    height: 35px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    border: none;
+    background: #36b37f;
+    margin: 0 10px 25px 0;
+    border-radius: 7px;
+    padding-bottom: 20px;
+  }
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please reduce your modal buttons padding because it still does not match with Figma design's modal buttons :+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

### Mobile version
| Your Design | Figma Design |
|--------|--------|
| ![Screenshot from 2023-06-29 18-22-53](https://github.com/Armynerh/personal_portfolio/assets/45226848/a184c9d8-173c-4517-b4fe-d574d9c120e1) | ![Screenshot from 2023-06-29 18-26-03](https://github.com/Armynerh/personal_portfolio/assets/45226848/1a70999c-1bfb-4e93-838b-2f7696d8c1a8) | 
### Desktop Version
| Your Design | Figma Design |
|--------|--------|
| ![Screenshot from 2023-06-29 18-27-14](https://github.com/Armynerh/personal_portfolio/assets/45226848/e1d6c27f-d101-4420-83eb-31d74ef96fd6) | ![Screenshot from 2023-06-29 18-25-46](https://github.com/Armynerh/personal_portfolio/assets/45226848/3308db80-77fc-4711-892f-c575a3cc136f) |

- [ ] Please increase the size of your icons as well :+1: